docs/content/dependencies.md

```diff
@@ -15,16 +15,30 @@ Here is a [full example][example] of a Porter manifest that uses dependencies.
 
 ## Define a dependency
 
-In the manifest, add entries for each dependency of your bundle. The key used is a short name for the dependent bundle that
+In the manifest, add entries for each dependency of your bundle. The `name` field takes a short name for the dependent bundle that
 you will use to reference the dependent bundle elsewhere in the bundle. For example you can reference the dependent bundle's
-outputs via `{{ bundle.dependencies.KEY.outputs }}`.
+outputs via `{{ bundle.dependencies.KEY.outputs }}`.  The `tag` field takes the bundle tag of the dependency.  Both `name` and `tag`
```

Comment:
Since the dependency now has a name, and we don't need to reference it by a map key, I think it would be helpful to to change how we mention it in our templating suggestion (below). I am only changing it here but I'm sure we use that form (and variations of it, so it would be good to search for `bundle.dependencies.` to check for them, like bundle.dependencies.DEP or ALIAS.

```suggestion
outputs via `{{ bundle.dependencies.NAME.outputs }}`.  The `tag` field takes the bundle tag of the dependency.  Both `name` and `tag`
```

Reply:
Oh good call, thank you.